### DISCUSSION TITLE
Package management

### DISCUSSION THREAD

**Discussion by dmaretskyi** [rome/tools#1417] [Q&A]
Are there any plans on integrating package management into Rome to replace yarn/npm? I'd love to see something similar to rust's cargo - where theres no separate install command and all of the dependencies are automatically resolved and installed if needed before building the package.

**Comment by ematipico** [comment]
Yes, we have plans to implement also the package manager aspect of the platform! It's not high priority at the moment, but it's in our roadmap!
